langchain4j-core/src/main/java/dev/langchain4j/model/chat/request/ChatRequest.java

```diff
@@ -16,7 +17,8 @@
 public class ChatRequest {
 
```

Comment:
Consider adding common/extra parameters in scope of this PR.
Since all LLM provider modules will have to adapt to the new API, it might make sense to update each module only once (adopt new API methods and model parameters) instead of in 2 steps.